Chaos/ChampionDrakathMerge.cs

```diff
@@ -0,0 +1,161 @@
+    public string[] MultiOptions = { "Generic", "Select" };
+    public string OptionsStorage = sAdv.OptionsStorage;
+    // [Can Change] This should only be changed by the author.
+    // If true, it will not stop the script if the default case triggers and the user chose to only get mats
+    private bool dontStopMissingIng = false;
+
+    public void ScriptMain(IScriptInterface Bot)
+    {
+        Core.BankingBlackList.AddRange(new[] { "Drakath Armor", "Champion Drakath Insignia", "Original Drakath Armor", "Blade of Chaos", "Chaos Avenger's Greatsword", "Chaos Avenger Armor", "Legendary Sword of Dragon Control", "The Supreme Arcane Staff", "Discordia Rose of Chaos", "Chaos Rose" });
+        Core.SetOptions();
+
+        BuyAllMerge();
+        Core.SetOptions(false);
+    }
+
+    public void BuyAllMerge(string? buyOnlyThis = null, mergeOptionsEnum? buyMode = null)
+    {
+        LOC.Hero();
+        //Only edit the map and shopID here
+        Adv.StartBuyAllMerge("championdrakath", 2055, findIngredients, buyOnlyThis, buyMode: buyMode);
+
+        #region Dont edit this part
+        void findIngredients()
+        {
+            ItemBase req = Adv.externalItem;
+            int quant = Adv.externalQuant;
+            int currentQuant = req.Temp ? Bot.TempInv.GetQuantity(req.Name) : Bot.Inventory.GetQuantity(req.Name);
+            if (req == null)
+            {
+                Core.Logger("req is NULL");
+                return;
+            }
+
+            switch (req.Name)
+            {
+                default:
+                    bool shouldStop = Adv.matsOnly ? !dontStopMissingIng : true;
+                    Core.Logger($"The bot hasn't been taught how to get {req.Name}." + (shouldStop ? " Please report the issue." : " Skipping"), messageBox: shouldStop, stopBot: shouldStop);
+                    break;
+                #endregion
+
+                case "Drakath Armor":
+                        DAB.DrakathArmorQuest();
+                    break;
+
+                case "Champion Drakath Insignia":
+                    Core.FarmingLogger(req.Name, quant);
+                    while (!Bot.ShouldExit && !Core.CheckInventory(req.Name, quant))
+                    {
+                        Core.Logger("You have to kill Champion Drakath weekly to get his insignias, use your army to kill it easily");
+                        return;
+                    }
+                    break;
+
+                case "Original Drakath Armor":
+                        DAB.DrakathOriginalArmor();
+                    break;
+
+                case "Blade of Chaos":
+                    Core.FarmingLogger(req.Name, quant);
+                    Core.EquipClass(ClassType.Solo);
+                    while (!Bot.ShouldExit && !Core.CheckInventory(req.Name, quant))
+                    {
+                        Core.HuntMonster("ultradrakath", "Champion of Chaos", req.Name, quant, false, false);
```

Comment:
false false is good 👍  now it needs done for the rest 🤣 of the hunts 👍  ty

Reply:
you can also keep the logger on the hunts if it's not inside a while (if it's inside a while statement it'll spam the logging if its not it'll just log it once)